exe/launchpad/launch_downscaling_gridding.R

```diff
@@ -39,12 +37,12 @@ RUNSUFFIX <- paste0( format( Sys.time(), '%m-%d-%H%M%S' ), '_',
 
 if ( DEBUG ) {
```

Comment:
@ssmithClimate Thanks, I think deleting the block is the best option. It was used for active development (calling the script from RStudio rather than command line), but is not too helpful for the public, and other developers may prefer other ways to test changes.